### ISSUE TITLE
Two iOS devices connected to the same Amnezia-WG server cannot make Whatsapp/Facetime call to each other

### ISSUE THREAD

**Issue by pvishnyakov** [amnezia-vpn/amnezia-client#400]
Cannot make whatsapp/facetime call between two iOS mobile devices connected to the same Amnezia vpn server via Amnezia-WG, traffic is dropped.

How to reproduce:
1. Buy VPS;
2. Install Amnezia client on 1st iOS device;
3. Install Amnezia server using Amnezia-WG;
4. Share connection settings to file;
5. Set up 2nd Amnezia iOS client using settings file;
6. Connect both clients to the vpn server;
7. Try make a call using whatsapp or facetime.

**Comment by dr-clinker** [comment]
+, if I share my connection with other device i cant use vpn on both devices at the same time.
Protocol: AWG
version: 4.1.1.0 (Nov 7 2023)

**Comment by pokamest** [comment]
Are you trying to use single config on several devices?
You need to generate new config (using share connection) for each device

**Comment by pvishnyakov** [comment]
> Are you trying to use single config on several devices? You need to generate new config (using share connection) for each device

You’re right, I was doing exactly this, didn’t know that new config is required for every new device. Fixed, tested, now it works, thanks for your help!
